dev/pom.xml

```diff
@@ -23,10 +23,15 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <name>CRISTAL-iSE Development</name>
-    <groupId>org.cristalise</groupId>
+    
+    <parent>
+        <groupId>org.cristalise</groupId>
+        <artifactId>cristalise</artifactId>
+        <version>4.0.0-SNAPSHOT</version>
```

Comment:
Would it be possible to remove the version tag form the module poms? The version should only be set in the parent pom.